rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/nodes.stats/30_discovery.yml

```diff
@@ -40,3 +40,99 @@
   - gte: { nodes.$master.discovery.published_cluster_states.incompatible_diffs: 0 }
   - gte: { nodes.$master.discovery.published_cluster_states.compatible_diffs: 0 }
   - is_false:  nodes.$master.roles
+
+---
+"Master timing stats":
```

Comment:
Really just asserting that the stats really do come out of the API and that we do the right thing with `?human`.